stdlib/bool.mli

```diff
@@ -0,0 +1,65 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                         The OCaml programmers                          *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Boolean values.
+
+    @since 4.08 *)
+
+(** {1:bools Booleans} *)
+
+type t = bool
+(** The type for boolean values. *)
+
+val not : bool -> bool
+(** [not b] is the boolean negation of [b]. *)
+
+val negate : ('a -> bool) -> ('a -> bool)
+(** [negate sat v] is [not (sat v)]. *)
```

Comment:
What does "sat" stand for? It's quite non-obvious.

Reply:
`sat`isfy, replace by `p`redicate (like in the `List` module).

Reply:
Okay, I find this utterly incomprehensible in context. If I was reading through this and looking for a function to use under these circumstances I would never hit on this one.